devices/xiaomi.js

```diff
@@ -898,6 +898,28 @@ module.exports = [
         },
         ota: ota.zigbeeOTA,
     },
+    {
+        zigbeeModel: ['lumi.motion.ac01'],
+        model: 'RTCZCGQ11LM',
+        vendor: 'Xiaomi',
+        description: 'Aqara Presence Detector FP1 (regions not supported for now)',
+        fromZigbee: [fz.aqara_opple],
+        toZigbee: [tz.RTCZCGQ11LM_presence, tz.RTCZCGQ11LM_monitoring_mode, tz.RTCZCGQ11LM_approach_distance],
+        exposes: [e.presence().withAccess(ea.STATE_GET), exposes.text('presence_event', ea.STATE).withDescription('Presence ' +
```

Comment:
Does it make sense to use `occupancy` instead of `presence`?

Reply:
I used `presence` because _RTCZCGQ11LM_ is functionally analogous to [TuYa MIR-HE200-TY](https://www.zigbee2mqtt.io/devices/MIR-HE200-TY.html), i.e. it defines the presence of motionless people.